compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.h

```diff
@@ -56,7 +56,7 @@ std::unique_ptr<OperationPass<func::FuncOp>> createLLVMCPUPeelPass();
 
 /// Pass to perform SplitReduction transformations of `LinalgOp`s.
 std::unique_ptr<OperationPass<func::FuncOp>>
-createLLVMCPUSplitReductionPass(bool enableReassociateFpReductions = false);
+createLLVMCPUSplitReductionPass(bool enableReassociateFpReductions = true);
```

Comment:
Not sure how this change is related to this patch.

Reply:
I think we should remove the default value. Enabling or disabling fp reassoc has significant consequences that every caller should think about.

Reply:
Reverted this.